app/controllers/app_groups_controller.rb

```diff
@@ -86,6 +102,19 @@ def create
       app_group_params[:labels] = {}
     end
 
+    # not sure if needed
```

Comment:
do we still need these line of codes? if no, we can delete it

Reply:
removed